### DISCUSSION TITLE
Blacklist ntfs3 module by default

### DISCUSSION THREAD

**Discussion by Unavailium** [linuxmint/discussions#680] [Ideas]
With reference to this thread: [viewtopic.php?t=434383](https://forums.linuxmint.com/viewtopic.php?t=434383) , I would like to suggest that the Mint devs consider making an update that blacklists the ntfs3 driver for everyone. It is clearly causing many problems, especially for newcomers from a Windows environment.

I'm not sure if that's seen as desirable by the Mint devs, or even if it's reasonably possible, but I think it should be considered.

**Comment by Hawaiihemd** [comment]
IMHO Mint 22.1 should include a config file in /etc/modprobe.d/ that blacklists ntfs3. In the release notes, people can be told about this and be explained to just delete the file if they like to use ntfs3.

I believe this ntfs3 situation is hard to grasp for people who have just built up enough courage to make the change over to a Linux OS.

**Comment by morbius1gh** [comment]
Please note that the blacklisting of ntfs3 does not impact the mounting of ntfs partitions. It only impacts which ntfs "driver" the file manager ( udisks2 ) uses. Per https://storaged.org/doc/udisks2-api/latest/mount_options.html#id-1.2.8.7 :+1: 

>  define order of filesystem driver priorities for the actual mount call,
>  required definition for non-matching driver names
> ntfs_drivers=ntfs3,ntfs

If ntfs3 is unavailable because it is blacklisted the fuse based ntfs ( ntfs-3g ) driver will be used.

Note: The suggestion in the release notes mentions using Disks to "repair filesystem". That may or may not resolve the immediate issue but if you continue to use ntfs3 it will eventually lead to corruption

**Comment by LinuxOnTheDesktop** [comment]
'[I]t will eventually lead to corruption': yikes.

**Comment by clefebvre** [comment]
Fixed in ubuntu-system-adjustments 2025.01.07.1-xia.
